src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -3,25 +3,26 @@
 import org.junit.Before;
 import org.junit.Test;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 public class DatabaseManagerTest {
-    DatabaseManager databaseManager;
+    private DatabaseManager databaseManager;
 
     @Before
     public void init() {
         databaseManager = new DatabaseManager();
     }
 
     @Test
-    public void testSuccessfulConnection() {
-        boolean expected = databaseManager.connect("sqlcmd", "sqlcmd", "sqlcmd");
-        assertEquals(expected, true);
+    public void testCorrectConnection() {
+        assertTrue(databaseManager.connect("sqlcmd", "sqlcmd", "sqlcmd"));
     }
 
     @Test
-    public void testUnSuccessfulConnection() {
-        boolean expected = databaseManager.connect("NotSqlcmd", "sqlcmd", "sqlcmd");
-        assertEquals(expected, false);
+    public void testConnectionWithWrongData() {
```

Comment:
Is it ok to write few asserts in one method if it tests exactly one functionality? I was thinking to separate these asserts, but finlly decide to leave them in one test method

Reply:
Try to name methods with original method name. For example:
testConnectWhen...Returns...()
or without 'test' word
connectWhen...ExecutedCorrectly()

About your question:
I think on these step it is needlessly.
You can create methods such as: 
connectWhenDatabaseNameIsIncorrectReturnsFalse()
connectWhenUserNameIsIncorrectReturnsFalse()
connectWhenPasswordIsIncorrectReturnsFalse()